src/message/types.ts

```diff
@@ -0,0 +1,74 @@
+import { SequenceNumber, ENR } from "../enr";
+
+export type RequestId = bigint;
```

Comment:
why do you treat id as number isn't that always 32 random bytes?

Maybe I'm missing something, but it seems like silly reason to drop safari and node 8 as there is no need for bigint here?!

Reply:
The spec doesn't define the length :crying_cat_face: 
We're using bigints elsewhere already in this spec (ENR sequence numbers).

It makes sense to use bigints here because the main things we do with requestIds are: storing request metadata keyed by requestId and doing comparisons with other requestIds, both much more suitable for bigint.

Reply:
I would still argue that hex encoded byte string would still be better than dropping node 8 and safari :thinking:

Reply:
Discv5 is a udp protocol that by design does not support the browser in its current state.
Node 8 reaches the end of maintenance on Dec. 31 2019.
Eth2 is a blockchain protocol that won't ramp up to full production for ~1 year.
Given those constraints, imo, it makes sense to use future-leaning features including bigint.